pycheribuild/projects/cross/cheribsd.py

```diff
@@ -679,6 +995,18 @@ def clean(self) -> ThreadJoiner:
         else:
             return self.async_clean_directory(builddir)
 
+    def _list_kernel_configs(self):
+        """Emit a list of valid kernel configurations that can be given as --kernel-config overrides"""
+        conf_dir = self.source_dir / "sys" / self.target_info.freebsd_target_arch_dir / "conf"
+        configs = conf_dir.glob("*")
+        blacklist = ["NOTES", "LINT", "DEFAULTS"]
+        self.info("Valid kernel configuration files for --freebsd/kernel-config:")
```

Comment:
```suggestion
        self.info("Valid kernel configuration files for --" +self.target + "/kernel-config:")
```

Reply:
Or even better since it ensures the second part won't become stale:
```suggestion
        self.info("Valid kernel configuration files for", self.get_config_option_name("kernel_config"))
```